app/experimenter/docs/adrs/0006-doc-hub-repo.md

```diff
@@ -0,0 +1,98 @@
+# Location for the new Experiment "Docs Hub" codebase + docs
+
+- Status: **draft**
+- Deciders: Kate Hudson, Jared Lockhart, Lauren Zugai, Tim Smith, Les Orchard, Jody Heavener
+- Date: 2021-03-12
+
+## Context and Problem Statement
+
+ADR #5, ["Use Docusaurus + GH Pages for the Nimbus User Doc Hub"](https://github.com/mozilla/experimenter/blob/main/app/experimenter/docs/adrs/0005-doc-hub.md), determined that our approach to setting up a new central location for Experimenter/Nimbus and related project documentation would use [Docusaurus](https://v2.docusaurus.io/), a static site generator, with the static files being served from [GitHub Pages](https://pages.github.com/). While we knew we would be introducing a new set of code we didn't explicitly decide where it would live and what the process for deploying would look like, beyond suggesting it could live in the existing Experimenter repository.
+
+In investigating how to set all of this up we realized there would be certain caveats to hosting and deploying this code from the Experimenter repository, and this lead to the consideration of using a dedicated repository. This ADR outlines our options and end decision.
+
+## Decision Drivers
+
+- How much setup work is involved, both for the code _and_ for integrations or other non-code processes?
+- How maintainable is this setup, both now and in years to come?
+- Not everyone who contributes to these docs is a programmer; how much of a learning curve does this present?
+
+## Considered Options
+
+- Use the Experimenter repository, main branch
+- Use the Experimenter repository, dedicated branch
+- Set up a new, dedicated repository
+
+## Decision Outcome
+
+Undecided.
```

Comment:
You can update this to `Set up a new, dedicated repository`, IMO this should be a true record since the discussion already happened